utam-core/src/main/java/utam/core/framework/context/MobilePlatformType.java

```diff
@@ -49,10 +49,10 @@ public boolean isAndroid() {
    */
   public static MobilePlatformType fromDriver(WebDriver driver) {
     if (driver instanceof AndroidDriver) {
-      return ANDROID;
+      return isTablet(driver) == true ? ANDROID_TABLET : ANDROID_PHONE;
```

Comment:
== true is redundant. method already returns boolean

Reply:
updated